src/js/controller/controller.js

```diff
@@ -782,7 +782,13 @@ define([
             this.resize = _view.resize;
             this.getSafeRegion = _view.getSafeRegion;
             this.setCues = _view.addCues;
-            this.setCaptions = _view.setCaptions;
+            this.setCaptions = function(stylingOptions) {
+                const provider = this.getProvider();
+                if (provider.name === 'chromecast') {
+                    _model.styleCaptions(stylingOptions);
```

Comment:
The view passes the options to `captionsrenderer.js`, which extends the defaults with the custom options and stores the modified options in the model. (See: https://github.com/jwplayer/jwplayer/blob/master/src/js/view/captionsrenderer.js#L173).

Let's discuss improving this.